src/LuaTransformer.ts

```diff
@@ -3928,6 +3966,57 @@ export class LuaTransformer {
         }
     }
 
+    public transformLuaTableCallExpression<T extends boolean>(
+        expression: ts.CallExpression,
+        isWithinExpressionStatement: T
+    ): T extends true ? tstl.Statement : tstl.Expression;
+    public transformLuaTableCallExpression(
+        expression: ts.CallExpression,
+        isWithinExpressionStatement: boolean
+    ): tstl.Statement | tstl.Expression {
+        const method = expression.expression as ts.PropertyAccessExpression;
+        const methodName = method.name.escapedText;
+        const signature = this.checker.getResolvedSignature(expression);
+        const tableName = (method.expression as ts.Identifier).escapedText;
+        const luaTable = tstl.createIdentifier(tableName);
+        const params = this.transformArguments(expression.arguments, signature);
+
+        const getIndexExpression = () => {
+            if (expression.arguments.length === 0) {
+                throw TSTLErrors.ForbiddenLuaTableUseException(
+                    expression,
+                    "A parameter is required for set() or get() on a '@LuaTable' object."
+                );
+            } else {
+                return tstl.createTableIndexExpression(luaTable, params[0], expression);
```

Comment:
What if `set` receives more than 2 parameters, or `get` receives more than 1?

Reply:
I was thinking this might be useful if something like `t.prop = 1, 0` needed to exist but I can't think of a use. `set()` now needs two parameters and `get()` can only take one